dashboard/app/models/levels/blockly.rb

```diff
@@ -456,15 +456,15 @@ def get_localized_property(property_name)
     if should_localize? && try(property_name)
       I18n.t(
         name,
-        scope: [:data, property_name.pluralize],
+        scope: [:data, property_name],
```

Comment:
This seems like it has the potential to break a lot of _other_ translations. I don't remember exactly why we used to pluralize here; why are we confident this is no longer needed?

Reply:
I checked `dashboard` folder and found only three uses of this method:

1. `failure_message_overrides`: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/levels/blockly.rb#L467
2. `long_instructions`: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/levels/blockly.rb#L471C59-L471C76
3. `short_instructions`: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/levels/blockly.rb#L567C50-L567C68

Since `long_instructions` and `short_instructions` are already plural, we shouldn't have any issues